packages/shared/src/hooks/streaks/useReadingStreak.ts

```diff
@@ -2,26 +2,29 @@ import { useQuery } from '@tanstack/react-query';
 import { generateQueryKey, RequestKey, StaleTime } from '../../lib/query';
 import { getReadingStreak, UserStreak } from '../../graphql/users';
 import { useAuthContext } from '../../contexts/AuthContext';
-import { useStreakExperiment } from './useStreakExperiment';
+import { useActions } from '../useActions';
+import { ActionType } from '../../graphql/actions';
 
 interface UserReadingStreak {
   streak: UserStreak;
   isLoading: boolean;
-  isEnabled: boolean;
+  shouldShowPopup: boolean;
 }
 
 export const useReadingStreak = (): UserReadingStreak => {
   const { user } = useAuthContext();
-  const { shouldShowStreak } = useStreakExperiment();
   const { data: streak, isLoading } = useQuery(
     generateQueryKey(RequestKey.UserStreak, user),
     getReadingStreak,
-    { enabled: shouldShowStreak, staleTime: StaleTime.Default },
+    { enabled: true, staleTime: StaleTime.Default },
```

Comment:
We should maybe check if the user opted out of weekly goal. If I remember correctly, we will use it for users to opt out. Probably better to check again with the product team.

Reply:
Will check with product about this, since here at least we were not taking it in consideration anyways

Reply:
product thread here: https://dailydotdev.slack.com/archives/C05TQL6HWTH/p1715694222530479

Anyways, I don't think this should be blocking since even if we want a similar toggle for reading streaks, keeping the same one with the weekly goal naming could bring complications. I suggest we remove and then product can decide if they want to introduce something like this as a follow up.

Reply:
Also yes we seem to use this for the moment, maybe the best thing to do is to actually work on renaming this into something reading streaks related

Reply:
Yep, sure thing. Thank you for checking.

Reply:
enabled property is true by default, so I think we can just omit it now 
```suggestion
    { staleTime: StaleTime.Default },
```